recipes/pyglotaran/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "pyglotaran" %}
+{% set version = "0.1.0" %}
+{% set download_name = "pyglotaran" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ download_name[0] }}/{{ download_name }}/{{ download_name }}-{{ version }}.tar.gz"
+  sha256: ec8d32208af1366f0a10a9adb1ad6644bf009bace8eaca86769218968bd240b9
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - glotaran=glotaran.cli.main:glotaran
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytest-runner
+  run:
+    - python >=3.6
+    - pytest-runner
+    - click >=7.0
+    - cloudpickle >=1.2
+    - dask[bag] >=2.8
+    - lmfit >=0.9.13,<1.0.1
+    - netCDF4 >=1.5.3
+    - numba >=0.48
+    - numpy >=1.17.3
+    - pandas >=0.25.2
+    - pyyaml >=5.2
+    - scipy >=1.3.2
+    - setuptools >=41.2
```

Comment:
do you need setuptools at runtime

Reply:
It's a dependency of xarray
https://github.com/pydata/xarray/blob/b9e6a36ff7a0ca3593165cf191f4152666fa4a66/setup.cfg#L79
we had problems being resolved properly with in the past, why we explilitly added to our setup.py .
https://github.com/glotaran/pyglotaran/issues/354
Since the xarray-feedstock isn't setting the minimum version, I'm not sure if we might get that problem again.
https://github.com/conda-forge/xarray-feedstock/blob/1cf08c9f60afb05d91f0f2cafc0ea664bf136e0e/recipe/meta.yaml#L26